serverless.yml

```diff
@@ -654,6 +654,62 @@ functions:
               - name: request.querystring.max_level
               - name: request.header.origin
 
+  node_api_nc_state:
+    handler: handlers/node_api.nc_state
+    maximumRetryAttempts: 0
+    package:
+      patterns:
+        - 'handlers/node_api.py'
+    layers:
+      - { Ref: PythonRequirementsLambdaLayer }
+    events:
+      - http:
+          path: node_api/nc_state
+          method: get
+          cors: true
```

Comment:
Should we add a comment about why this one has no caching?

Reply:
Done https://github.com/HathorNetwork/hathor-explorer-service/pull/286/commits/ee73a81c0239a77ed6010df6de7ae47619f61cf4